tests/characters/control/test_web_control.py

```diff
@@ -93,15 +77,15 @@ def test_alice_character_control_revoke(alice_control_test_client, federated_bob
         'n': 3,
         'expiration': (maya.now() + datetime.timedelta(days=3)).iso8601(),
```

Comment:
Not at all IPC-specific (many other places also), but I think we may want to test if the API works correctly when we happen to have `0`-th microsecond. One can do that as `(maya.MayaDT(maya.now().epoch) + datetime.timedelta(days=3)).iso8601()`. The result would be something like `2019-06-20T03:07:16Z` (which we'll happen to have one time in million!). Would the tests still pass?